xwiki-platform-core/xwiki-platform-test/xwiki-platform-test-ui/src/main/java/org/xwiki/test/ui/po/BasePage.java

```diff
@@ -67,10 +67,10 @@ public class BasePage extends BaseElement
     @FindBy(id = "contentmenu")
     private WebElement contentMenuBar;
 
-    @FindBy(xpath = "//div[@id='tmCreate']/a[contains(@role, 'button')]")
+    @FindBy(xpath = "//div[@id='tmCreate']//a")
```

Comment:
Why did you add this double slash? The location of the link shouldn't have changed.

Reply:
My bad, I misunderstood when looking at doc for the xpath syntax, I just assumed the // was direct child and the / was any descendant.
I'll fix this when solving the merge issues with #2171 :+1: